deploy/charts/values.yaml

```diff
@@ -29,6 +29,8 @@ kube-prometheus-stack:
   prometheus-node-exporter:
     rbac:
       pspEnabled: false
+    service:
+      port: 10100
```

Comment:
Fixed.
I can leave it empty and it will default to 9100
It should be exposed for configuration so that it can be  set to alternate values at helm install time.